### ISSUE TITLE
which-key integration

### ISSUE THREAD

**Issue by jthaman** [abo-abo/ace-window#201]
It would be nice if ace-window automatically integrated with which-key to show the dispatch-alist when invoked.

**Comment by hammerandtongs** [comment]
Yeah I just noticed I really wanted to swap windows but didn't remember that ace-window could even do that, which-key would be really great to expose all the great things ace-window can do.

**Comment by scmbradley** [comment]
+1

**Comment by OldhamMade** [comment]
It would be great to get this integration (or maybe a `defadvice` workaround in the wiki?) so that we can be reminded of the options when using `ace-window`.

**Comment by ccarlile** [comment]
I want to point out the relatively-obscure `aw-show-dispatch-help`, which pops up a help window in the minibuffer and starts the window-selection process. I replaced my `M-o` binding with it. You can also type `?` after invoking `ace-window` to show the help
